web/src/common/ProjectStorage.ts

```diff
@@ -1,4 +1,5 @@
 import type { FeatureCollection, MultiPolygon, Polygon } from "geojson";
+import type { AppFocus } from "../stores";
```

Comment:
This is an aside, but I recall hitting problems that I ultimately blamed on circular import references. 

The error was something like "`new Database()` is not a constructor", and my resolution was to make the imports more DAG like.

IIRC importing stores into ProjectStorage was one of the cases that tripped me up before. But this seems to be building fine, so now I'm confused.  Any idea why?

Reply:
It's a good point generally -- we could maybe have a `type.ts` with no dependencies itself. I suspect this is fine because it's a TS type definition import only?